frontend/src/component/personalDashboard/Grid.tsx

```diff
@@ -1,6 +1,74 @@
 import { Box, Grid, styled } from '@mui/material';
 import type { Theme } from '@mui/material/styles/createTheme';
 
+export const ContentGridContainer = styled('div')({
+    containerType: 'inline-size',
+});
+
+const ContentGrid2 = styled('article')(({ theme }) => {
+    return {
+        backgroundColor: theme.palette.divider,
+        borderRadius: `${theme.shape.borderRadiusLarge}px`,
+        overflow: 'hidden',
+        border: `0.5px solid ${theme.palette.divider}`,
+        gap: `2px`,
+        display: 'grid',
+
+        '&>*': {
+            backgroundColor: theme.palette.background.paper,
+        },
+    };
+});
+
+export const ProjectGrid = styled(ContentGrid2)(({ theme }) => ({
+    gridTemplateAreas: `
+            "title"
+            "onboarding"
+            "projects"
+            "box1"
+            "box2"
+            "owners"
+            `,
+
+    '@container (min-width: 1000px)': {
+        gridTemplateColumns: '1fr 1fr 1fr',
+        gridTemplateAreas: `
+                "title onboarding onboarding"
+                "projects box1 box2"
+                ". owners owners"
+            `,
+    },
+
+    '@supports not (container-type: inline-size)': {
+        [theme.breakpoints.up('lg')]: {
+            gridTemplateColumns: '1fr 1fr 1fr',
+            gridTemplateAreas: `
+                "title onboarding onboarding"
+                "projects box1 box2"
+                ". owners owners"
+            `,
+        },
+    },
+}));
+
+export const SpacedGridItem2 = styled('div')(({ theme }) => ({
+    padding: theme.spacing(4),
+}));
+
+export const EmptyGridItem = styled('div')(({ theme }) => ({
+    display: 'none',
+
+    '@container (min-width: 1000px)': {
+        display: 'block',
+    },
+
+    '@supports not (container-type: inline-size)': {
+        [theme.breakpoints.up('lg')]: {
+            display: 'block',
+        },
+    },
+}));
```

Comment:
This is an empty element to take up spaces in the grid when it's multi-column (the special `.` area). We hide it to avoid showing an empty div in the list when we collapse the grid.